tests/suites/deploy/deploy_charms.sh

```diff
@@ -213,7 +213,7 @@ run_deploy_lxd_to_container() {
 		attempt=$((attempt + 1))
 		if [ $attempt -eq 10 ]; then
 			# shellcheck disable=SC2046
-			echo $(red "timeout: waiting for lxc profile to show 50sec")
+			echo $(red "timeout: waiting for removal of lxc profile 50sec")
```

Comment:
why this change?

Reply:
Ian, I suppose tests run_deploy_lxd_to_machine and run_deploy_lxd_to_container check the same issue. If so, in the second "while-cycle" we need to ensure that the old profile is removed, and I've just fixed the error message to to reflect it.